src/components/App.jsx

```diff
@@ -54,16 +63,53 @@ class App extends React.Component {
       });
   };
 
+  /**
+   * handles clicking the "see more" text in the creator component
+   */
+  handleSeeMoreClick() {
+    const { modalView } = this.state;
+    this.toggleModalView();
+  }
+
+  /**
+   * toggles this.state.modalView
+   */
+  toggleModalView() {
+    const { modalView } = this.state;
+    this.setState({ modalView: !modalView });
+  }
+
+  /**
+   * checks if this.state.modalView is true.
+   * if so, renders modal to page
+   */
+  renderModal() {
+    const { modalView, creator, collaborators } = this.state;
+    if (modalView) {
+      return (
+        <div>
+          <WholePageWrapper onClick={() => {this.toggleModalView()}} >
+          </WholePageWrapper>
+          <ModalWrapper onClick={() => {this.toggleModalView()}} >
+            <CreatorModal
+              creator={creator}
+              collaborators={collaborators} />
+          </ModalWrapper>
+        </div>
+      );
+    } else {
+      return null;
+    }
+  }
+
   render() {
-    const { id, listingTitle, pledges, creator, collaborators } = this.state;
-    console.log('this.state: ', this.state);
+    const { id, listingTitle, pledges, creator, collaborators, modalView } = this.state;
     return (
       <div className="pledgeComponent AppComponentDiv">
-        <Title>
-          {listingTitle}
-        </Title>
-        <CreatorWrapper>
-          <Creator creator={creator} />
+        <CreatorWrapper onClick={() => {this.handleSeeMoreClick()}} >
+          <Creator
+            creator={creator}
+            handleSeeMoreClick={this.handleSeeMoreClick} />
         </CreatorWrapper>
```

Comment:
It looks to me like you're putting a click handler on both the wrapper and the inner element, either of which call the same function. Maybe worth thinking about whether this is necessary. 

In the same vein, it looks like CreatorWrapper just applies styling. This could potentially be simplified by making CreatorWrapper a styled Creator instead of a styled div, or building the Creator component with a styled div and eliminating the wrapper altogether. Nothing would change functionally, but it would just make it a little more concise.